docs/specs/moniker.yml

```diff
@@ -179,25 +179,24 @@ inputs:
 outputs:
   9d4e15fd/docs/a.json: |
     {
-      "document_id": "2f5db941-5d87-5c7e-668b-00a1e364cc0e",
+      "document_id": "e6bc2b27-51df-e84a-768d-3008b0540975",
       "content": "<p>Moniker: netcore-1.0, netcore-1.1</p>",
       "monikers": ["netcore-1.0", "netcore-1.1"]
     }
-  9d4e15fd/docs/a-old.json: |
-    {
-      "redirect_url": "/docs/a",
-      "monikers": ["netcore-1.0", "netcore-1.1"]
-    }
   f961cd88/docs/a.json: |
     {
-      "document_id": "2f5db941-5d87-5c7e-668b-00a1e364cc0e",
+      "document_id": "977a62e2-6380-388e-e785-819686caac32",
       "content": "<p>Moniker: netcore-1.2, netcore-1.3</p>",
       "monikers": ["netcore-1.2", "netcore-1.3"]
     }
-  f961cd88/docs/a-old.json: |
+  .publish.json: |
     {
-      "redirect_url": "/docs/a",
-      "monikers": ["netcore-1.2", "netcore-1.3"]
+      "publish": [
+        { "url": "/docs/a-old", "redirect_url": "/docs/a1", "monikers": [ "netcore-1.0", "netcore-1.1" ] },
```

Comment:
do we have output path here? why need "publish" in ".publish.json"?

Reply:
yes, `path` is checked in other PRs, this test focuses on `redirect_url`.

Reply:
what about the second question?

Reply:
Maybe to avoid making the root level become array type